src/abstracts/SablierV2ProxyTarget.sol

```diff
@@ -167,6 +167,39 @@ abstract contract SablierV2ProxyTarget is
         lockup.withdrawMultiple(streamIds, to, amounts);
     }
 
+    /// @inheritdoc ISablierV2ProxyTarget
+    function withdrawMultipleToRecipient(
```

Comment:
Can we just keep naming this `withdrawMultiple`? Overloading functions is fine.

Reply:
It wouldn't be fine, because someone integrating a target wouldn't be able to use:

```solidity
abi.encodeWithSelector(ISablierV2ProxyTarget.withdrawMultiple.selector, streamIds, to, amounts);
abi.encodeCall(ISablierV2ProxyTarget.withdrawMultiple, (lockup, streamIds, to, amounts))
```

Because it will get this error:

<img width="534" alt="image" src="https://github.com/sablier-labs/v2-periphery/assets/99738872/b70e47ca-59e1-4b22-9f76-8ad5497d0791">

So we should have different names.

Reply:
Ok fine. But in this case let's use plural "recipients" instead of "recipient".